test/jasmine/tests/ternary_test.js

```diff
@@ -12,6 +12,16 @@ var doubleClick = require('../assets/double_click');
 var customMatchers = require('../assets/custom_matchers');
 
 
+function getClientPosition(selector, index) {
```

Comment:
@n-riesco Nice helper!

Would you mind making this an assets module in `test/jasmine/assets/get_client_position.js`?